util/ui/ui.go

```diff
@@ -274,6 +276,27 @@ func (ui *UI) DisplayTextWithFlavor(template string, templateValues ...map[strin
 	fmt.Fprintf(ui.Out, "%s\n", ui.TranslateText(template, firstTemplateValues))
 }
 
+// DisplayJSON encodes and indents the input
+// and outputs the result to ui.Out.
+func (ui *UI) DisplayJSON(name string, jsonData interface{}) error {
+	buff := new(bytes.Buffer)
```

Comment:
From the [docs](https://golang.org/pkg/bytes/#example_Buffer)
```go
var b bytes.Buffer // A Buffer needs no initialization.
```
While `new()` works, I think just declaring a var is simpler, and reminds people that this is not C++.

Reply:
Actually do you see any problem with using the the ` ui.Out` as a Writer and removing the new buffer?
`encoder := json.NewEncoder(ui.Out)`